### PR TITLE
Fixed contact dialog box reappearing

### PR DESCRIPTION
**Pull request by alecdev105**

### CODE REVIEW

File: src/Components/ImportContactsDialog/ImportContactsDialog.jsx [hunk 1]
```diff
@@ -109,9 +109,12 @@ const ImportContactsDialog = ({ status, callback, onImport }) => {
 
             var all = document.getElementsByClassName("initial__modal");
             for (var i = 0; i < all.length; i++) {
-              all[i].style.display = "block";
+              all[i].style.display = "none";
             }
 
+            // Mimicing close button click
+            document.getElementsByClassName("btn-close")[0].click();
```
Comment (biswanathsahoo) [inline src/Components/ImportContactsDialog/ImportContactsDialog.jsx:116]: Please use state or reducer to close the dialog box instead of this

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

File: src/Components/ImportContactsDialog/ImportContactsDialog.jsx (hunk 1 above)
Comment (alecdev105) [reply src/Components/ImportContactsDialog/ImportContactsDialog.jsx]: @biswanathsahoo Updated it

**Comment by biswanathsahoo** [comment]
@alecdev105 can you please resolve the conflict

File: src/Components/ImportContactsDialog/ImportContactsDialog.jsx [hunk 2]
```diff
@@ -109,9 +111,12 @@ const ImportContactsDialog = ({ status, callback, onImport }) => {
 
             var all = document.getElementsByClassName("initial__modal");
             for (var i = 0; i < all.length; i++) {
-              all[i].style.display = "block";
+              all[i].style.display = "none";
             }
 
+            // Open Create NFT Dialog
+            dispatch({ type: "createnft__open" });
```
Comment (biswanathsahoo) [inline src/Components/ImportContactsDialog/ImportContactsDialog.jsx:118]: This should be conditional, if someone is importing the contact for the 1st time after login, then we have to open create NFT popup, else if someone is accessing this after creating the NFT then we should open the contact list popup. But, this line will always open the create NFT popup after importing the contact.

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

File: src/Components/ImportContactsDialog/ImportContactsDialog.jsx (hunk 2 above)
Comment (alecdev105) [reply src/Components/ImportContactsDialog/ImportContactsDialog.jsx:118]: Ok

Comment (alecdev105) [reply src/Components/ImportContactsDialog/ImportContactsDialog.jsx]: @biswanathsahoo Resolved conflict issue.

**Pull request merged**